src/serac/physics/benchmarks/benchmark_serac_thermal_functional.cpp

```diff
@@ -0,0 +1,191 @@
+// Copyright (c) 2019-2021, Lawrence Livermore National Security, LLC and
```

Comment:
This is copied from physics/tests/serac_thermal_functional.cpp. In order to avoid code duplication for benchmarks, it probably makes sense to just do benchmarks for integrated tests and reuse the input decks.

Reply:
This will be considered in a follow on pull request.